eland/etl.py

```diff
@@ -526,29 +526,17 @@ def csv_to_eland(  # type: ignore
 
     first_write = True
     for chunk in reader:
-        if first_write:
-            pandas_to_eland(
-                chunk,
-                es_client,
-                es_dest_index,
-                es_if_exists=es_if_exists,
-                chunksize=chunksize,
-                es_refresh=es_refresh,
-                es_dropna=es_dropna,
-                es_type_overrides=es_type_overrides,
-            )
-            first_write = False
-        else:
-            pandas_to_eland(
-                chunk,
-                es_client,
-                es_dest_index,
-                es_if_exists="append",
-                chunksize=chunksize,
-                es_refresh=es_refresh,
-                es_dropna=es_dropna,
-                es_type_overrides=es_type_overrides,
-            )
+        pandas_to_eland(
```

Comment:
This is super great! :muscle: 

One nit is that we should add a comment on the `es_if_exists` line (`# es_if_exists should be 'append' except on the first call to pandas_to_eland()`?) so it stands out from all the other lines. Maybe moving it to the bottom of the function call would help as well with readability.